src/curl.c

```diff
@@ -150,16 +152,23 @@ static int check_connection(const char *test_capath)
 	case CURLE_OK:
 		return 0;
 	case CURLE_SSL_CACERT:
-		warn("Curl - Unable to verify server SSL certificate\n");
+		debug("Curl - Unable to verify server SSL certificate\n");
 		return -SWUPD_BAD_CERT;
 	case CURLE_SSL_CERTPROBLEM:
-		warn("Curl - Problem with the local client SSL certificate\n");
+		debug("Curl - Problem with the local client SSL certificate\n");
 		return -SWUPD_BAD_CERT;
 	case CURLE_OPERATION_TIMEDOUT:
+		debug("Curl - Check connection Timed out\n");
```

Comment:
"Check connection"? it sounds strange, isn't it better to just use "connection timed out"? also drop the capital "T"

Reply:
I'll change to "check_connection - timed out". It's a debug message and it's good to know where is this from.

Reply:
No, we don't need that prefix. I'll leave just the string Time out